lib/cable_provider.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:convert';
+
+import 'package:action_cable/action_cable.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+import 'env_provider.dart';
+
+final cableProvider = Provider<ActionCable>((ref) {
```

Comment:
this feels pretty good as a `Provider` that just assumes `DotEnv.env` has already been loaded in `main.dart`, which is currently true

Reply:
Instead of depending on a dotenv provider? I'm guessing this is to avoid rebuilds or something?

Seems fine but a lil smelly no doubt.

Reply:
more like, depending on futures felt bad to me so I wanted to avoid it here if possible

maybe I should just get used to depending on futures though